test/remote/gateways/remote_orbital_test.rb

```diff
@@ -1232,7 +1243,13 @@ def setup
       tax_indicator: '1',
       tax: '75',
       purchase_order: '123abc',
-      zip: address[:zip]
+      zip: address[:zip],
```

Comment:
It's not super clear by looking at the diff, but these are in a separate Tandem Class that uses different credentials. When adding all fields to level_2_options, I got errors about fields being restricted to Canadian merchants which is the reason for the new remote test with that currency and those fields are only added in that test.